src/tests/ftest/io/ior_small.yaml

```diff
@@ -39,14 +40,14 @@ ior:
 #          F: "-v -W -w -r -z"
     transfersize_blocksize: !mux
         256B:
-            t: '256B'
-            b: '4M'
+            transfer_size: '256B'
+            block_size: '4M'
         1M:
-            t: '1M'
-            b: '32M'
+            transfer_size: '1M'
+            block_size: '32M'
     objectclass: !mux
-        oclass_Large:
-            o: "LARGE"
+        oclass_SX:
+            daos_oclass: SX
 # Uncomment when DAOS-2305 is resolved
-#        oclass_R2:
-#            o: "R2"
+#        oclass_RP_2G2:
+#            daos_oclass: "RP_2G2"
```

Comment:
Same as above. Lets use "RP_2GX" to use all the present targets.

Reply:
Fixed